lib/matplotlib/lines.py

```diff
@@ -978,16 +989,28 @@ def set_color(self, color):
 
     def set_drawstyle(self, drawstyle):
         """
-        Set the drawstyle of the plot
+        Set the drawstyle of the plot.
 
-        'default' connects the points with lines. The steps variants
-        produce step-plots. 'steps' is equivalent to 'steps-pre' and
-        is maintained for backward-compatibility.
+        The drawstyle determines how the points are connected.
 
         Parameters
         ----------
         drawstyle : {'default', 'steps', 'steps-pre', 'steps-mid', \
 'steps-post'}
+            Possible values:
+
+            - 'default': The points are connected with straight lines.
+            - 'steps-pre', 'steps-mid', 'steps-post': The points are
+              connected with step-like lines, i.e. horizontal lines with
+              vertical steps.
+              The postfix pre/mid/post determines the position of the vertical
+              step (step at the beginning of the line segment, i.e. the line
```

Comment:
the explanation starts becoming so long that it would perhaps read better as a nested list...

Reply:
New try. :smile: